src/EditorFeatures/CSharpTest/Structure/TypeDeclarationStructureTests.cs

```diff
@@ -211,6 +216,7 @@ public async Task TestStruct2(string typeKind)
         [Theory, Trait(Traits.Feature, Traits.Features.Outlining)]
         [InlineData("enum")]
         [InlineData("class")]
+        [InlineData("record")]
         [InlineData("struct")]
         [InlineData("interface")]
         public async Task TestStruct3(string typeKind)
```

Comment:
i don't really even get how this test is working.

Reply:
I'm not sure too. Just extended it for records for completeness.